ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java

```diff
@@ -7164,6 +7166,14 @@ protected Operator genFileSinkPlan(String dest, QB qb, Operator input)
     return output;
   }
 
+  private void commitInsert(Table table, boolean overwrite) {
+    Task<? extends Serializable>[] tasks = new Task[this.rootTasks.size()];
+    tasks = this.rootTasks.toArray(tasks);
+    InsertTableDesc insertCommitHook = new InsertTableDesc(table.getTTable(), overwrite);
```

Comment:
Change name of object, e.g., to _insertDesc_; in the future, the descriptor might be useful beyond commit.